README.md

```diff
@@ -28,17 +28,61 @@ Simple data loader for CGP HCA Data Store
 Because this program uses Amazon Web Services and Google Cloud Platform, you will need to set up credentials
 for both of these before you can run the program.
 
-### AWS credentials
+### AWS Credentials
 1. If you haven't already you will need to make an IAM user and create a new access key. Instructions are
    [here](https://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html).
 
 1. Next you will need to store your credentials so that Boto can access them. Instructions are
    [here](https://boto3.readthedocs.io/en/latest/guide/configuration.html).
 
-### GCP credentials
+### GCP Credentials
 1. Follow the steps [here](https://cloud.google.com/docs/authentication/getting-started) to set up your Google
    Credentials.
 
+## (Optional) Cloud Metadata Credentials Setup
+When the loader submits data, it actually needs access to the referenced files in the account to obtain metadata 
+(e.g. hash and size) that may be in another account.
+
+If the data is public, this is unnecessary.  However, if it is private data, a second account that is authorized 
+needs to be specified.
+
+### (Optional) GCP Metadata Credentials
+Make sure you have [gcloud](https://cloud.google.com/sdk/install) installed and run:
```

Comment:
Consider an introductory comment, like:
```
If GCP files are being loaded that specifically require Google user credentials (rather than Google Service Account credentials), perform the following steps:
```